clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/converters/servergroup/CloneKubernetesAtomicOperationConverter.groovy

```diff
@@ -33,6 +34,9 @@ class CloneKubernetesAtomicOperationConverter extends AbstractAtomicOperationsCr
   }
 
   CloneKubernetesAtomicOperationDescription convertDescription(Map input) {
-    KubernetesAtomicOperationConverterHelper.convertDescription(input, this, CloneKubernetesAtomicOperationDescription)
+    def converted = KubernetesAtomicOperationConverterHelper.convertDescription(input, this, CloneKubernetesAtomicOperationDescription)
+    converted.sourceCredentials = (KubernetesNamedAccountCredentials) this.getCredentialsObject(converted.source.account)
```

Comment:
Just wondering, why isn't this converted in `convertDescription`? If it was, we wouldn't need this extra handling